src/coreclr/hosts/inc/coreclrhost.h

```diff
@@ -46,5 +46,22 @@ CORECLR_HOSTING_API(coreclr_execute_assembly,
             unsigned int* exitCode);
 
 #undef CORECLR_HOSTING_API
-                      
+
+struct SequencePointInfo
+{
+    int lineNumber, ilOffset;
+    char16_t* fileName;
```

Comment:
Now that you have moved the stuff inside of the hosting api code, you can ignore this comment.